composer.json

```
@@ -96,7 +96,7 @@
         "orchestra/testbench-core": "^8.0",
```

```suggestion
        "orchestra/testbench-core": "dev-phpunit10",
```

We should use this branch for now (alias to `master`)